### PR TITLE
Dockerfile fix, Documentation updated

### PR DESCRIPTION
**Pull request by ruzickap**
Dockerfile fix, Documentation updated

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -36,3 +36,18 @@ Usage
 ------
 Execute ``linkchecker http://www.example.com``.
 For other options see ``linkchecker --help``.
+
+Docker usage
+-------------
+
+If you do not want to install any additional libraries/dependencies you can use Docker image.
```
Comment (mgedmin) [inline README.rst]: I'm not a native English speaker so I could be wrong, but I think there ought to be a "the" in front of "Docker image".

**Comment by mgedmin** [comment]
BTW while testing this I noticed a warning

```
WARNING linkcheck.check 2018-03-25 11:09:02,555 MainThread Running as root user; dropping privileges by changing user to nobody.
```

Can we do something about that in the Dockerfile?

File: README.rst (hunk 1 above)
Comment (ruzickap) [reply README.rst]: I'm not a native speaker too... sorry. 
Fixed...

**Comment by ruzickap** [comment]
I also fixed the "Warning" in my last commit...
Thanks for letting me know...

**Comment by anarcat** [comment]
it would be nice if docker wouldn't run linkchecker as root, to be honest... is there anything we can do about that?

**Comment by ruzickap** [comment]
Yes...
The `-u $(id -u):$(id -g)` parameter will ensure, that linkchecker will run as "non-root" user.
So if you use the "updated examples" from last commit - you should not see the "Running as root user" warning message...

**Comment by anarcat** [comment]
good enough, thanks

**Pull request merged**